api/test_resources/boxes.json

```diff
@@ -0,0 +1,1341 @@
+              "created-apps": [
+                {
+                  "created-at-round": 1,
+                  "deleted": false,
+                  "id": 3,
+                  "params": {
+                    "approval-program": "CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=",
+                    "clear-state-program": "CIEB",
+                    "creator": "N5T74SANUWLHI6ZWYFQBEB6J2VXBTYUYZNWQB2V26DCF4ARKC7GDUW3IRU",
+                    "global-state-schema": {
+                      "num-byte-slice": 0,
+                      "num-uint": 0
+                    },
+                    "local-state-schema": {
+                      "num-byte-slice": 0,
+                      "num-uint": 0
+                    }
+                  }
+                }
+              ],
+              "created-at-round": 0,
+              "deleted": false,
+              "pending-rewards": 0,
+              "reward-base": 0,
+              "rewards": 0,
+              "round": 8,
+              "sig-type": "sig",
+              "status": "Offline",
+              "total-apps-opted-in": 0,
+              "total-assets-opted-in": 0,
+              "total-box-bytes": 0,
+              "total-boxes": 0,
+              "total-created-apps": 1,
+              "total-created-assets": 0
+            },
+            {
+              "address": "OKUWMFFEKF4B4D7FRQYBVV3C2SNS54ZO4WZ2MJ3576UYKFDHM5P3AFMRWE",
+              "amount": 999999499000,
+              "amount-without-pending-rewards": 999999499000,
+              "created-apps": [
+                {
+                  "created-at-round": 1,
+                  "deleted": false,
+                  "id": 5,
+                  "params": {
+                    "approval-program": "CCABADEYQQB1NhoAgAZjcmVhdGUSQQAYgSAxG4ECEkAABUg2GgIXNhoBTLlEQgBONhoAgAZkZWxldGUSQQAINhoBvERCADc2GgCAA3NldBJBAAs2GgEiNhoCu0IAIDYaAIAFY2hlY2sSQQARNhoBIjYaAhW6NhoCEkRCAAEAgQE=",
+                    "clear-state-program": "CIEB",
+                    "creator": "OKUWMFFEKF4B4D7FRQYBVV3C2SNS54ZO4WZ2MJ3576UYKFDHM5P3AFMRWE",
+                    "global-state-schema": {
+                      "num-byte-slice": 0,
+                      "num-uint": 0
+                    },
+                    "local-state-schema": {
+                      "num-byte-slice": 0,
+                      "num-uint": 0
+                    }
+                  }
+                }
+              ],
+              "created-at-round": 0,
+              "deleted": false,
+              "pending-rewards": 0,
+              "reward-base": 0,
+              "rewards": 0,
+              "round": 8,
+              "sig-type": "sig",
+              "status": "Offline",
+              "total-apps-opted-in": 0,
+              "total-assets-opted-in": 0,
+              "total-box-bytes": 0,
+              "total-boxes": 0,
```

Comment:
Notice that this account which is _not an app account_ still provides box total summaries. Are we okay with this behavior?

@michaeldiamant @jannotti @winder

Reply:
(it's not possible -under the current design- to tell the difference between a non-app account and an app-account with no boxes)

Reply:
@tzaffi I think _yes_. 

Here are my thoughts - happy to discuss live if helpful.
* My understanding is that algod + indexer do _not_ differentiate between logical kinds of accounts (e.g. app account vs non-app account).
* Perhaps there's a desire to enrich the model because boxes are specific to a kind of account usage.  If so, my initial reaction is to keep as is for now.  I imagine specializing account representation involves _many_ concerns.